src-tauri/src/cropper/mod.rs

```diff
@@ -1,10 +1,18 @@
 use crate::AppState;
-use tauri::{AppHandle, Manager, Position, WebviewUrl, WebviewWindowBuilder};
+use tauri::{AppHandle, LogicalSize, Manager, WebviewUrl, WebviewWindowBuilder};
+use windows::Win32::UI::WindowsAndMessaging::{SetWindowDisplayAffinity, WINDOW_DISPLAY_AFFINITY};
```

Comment:
Shoud this `use` be moved inside the "windows" cfg? Similar to 'cocoa' for MacOS